share/wrf_timeseries.F

```diff
@@ -351,15 +351,20 @@ SUBROUTINE calc_ts( grid )
        ! FALSE to output T and Q at 2-m and wind at 10-m diagnostic levels:
    LOGICAL, PARAMETER :: ts_model_level = .FALSE. 
 
-   !Allocate the arrays for wind components
-#if ( EM_CORE == 1 )   
-   ALLOCATE ( earth_u_profile(grid%max_ts_level), earth_v_profile(grid%max_ts_level) )
-#endif
 
-   IF ( grid%ntsloc_domain .LE. 0 ) RETURN
+   IF ( grid%ntsloc_domain .LE. 0 ) THEN
+       RETURN
+   END IF
 
 #if ((EM_CORE == 1) && (DA_CORE != 1))
-   IF ( grid%dfi_opt /= DFI_NODFI .AND. grid%dfi_stage /= DFI_FST ) RETURN
+   IF ( grid%dfi_opt /= DFI_NODFI .AND. grid%dfi_stage /= DFI_FST ) THEN
+       RETURN
+   END IF
+#endif
+
+   !Allocate the arrays for wind components
+#if ( EM_CORE == 1 )   
+   ALLOCATE ( earth_u_profile(grid%max_ts_level), earth_v_profile(grid%max_ts_level) )
```

Comment:
@likeuclinux Are the changes made in the PR for wrf_timeseries.F completely the same as in PR-2139?

Reply:
@likeuclinux It looks like there is an extra empty line in the version of wrf_timeseries.F in this PR.